postman/src/services/persistence/LineaMessageDBService.ts

```diff
@@ -63,44 +61,20 @@ export class LineaMessageDBService extends MessageDBService implements IMessageD
    */
   public async getMessageToClaim(
     contractAddress: string,
-    _gasEstimationMargin: number,
+    gasEstimationMargin: number,
     maxRetry: number,
     retryDelay: number,
   ): Promise<Message | null> {
-    const feeEstimationOptions = await this.getClaimDBQueryFeeOptions();
+    const { maxFeePerGas } = await this.provider.getFees();
```

Comment:
iiuc, ihis `maxFeePerGas` is not from `linea_estimateGas` and so its value is supposed to be greater than what returned by `linea_estimateGas`, it means it would be restricter to pick the message to claim in db than how we determine a transaction is "underpriced" (we use `getBlockExtraData` which should be the same as calling `linea_estimateGas`), what's the motivation of that? why not both use the same gas price from `getBlockExtraData`  like before?